app/components/emoji_picker/emoji_picker.js

```diff
@@ -74,6 +77,10 @@ export default class EmojiPicker extends PureComponent {
             this.setState({
                 emojis: this.renderableEmojis(this.props.emojisBySection, nextProps.deviceWidth)
             });
+
+            if (this.refs.search_bar) {
+                this.refs.search_bar.blur();
```

Comment:
Why do we need to blur the search bar when rotating the device?

Reply:
it actually blurs but does not remove the `Cancel` button which makes it behave erroneously, by doing this we avoid the erroneous behavior